Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:2.3-slim
+MAINTAINER OpenStreetMap
+ENV REFRESHED_AT 2016-09-15
+
+# Install packages
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
These are different than the listed [minimum requirements](https://github.com/openstreetmap/openstreetmap-website/blob/master/INSTALL.md#minimum-requirements). I can understand ruby being different since it's coming from a ruby image and the lack of postgresql, but are there reasons for the other differences?